scripts/safe-commit-linter.sh

```diff
@@ -51,11 +55,31 @@ if [ -z "${MAIN_COMMIT+x}" ]; then
     MAIN_COMMIT="origin/master"
 fi
 
-merge_base=`git merge-base "$current_branch_hash" "$MAIN_COMMIT"`
-diff_files=`git diff --name-only "$current_branch_hash" "$merge_base"`
+merge_base_command="git merge-base $current_branch_hash $MAIN_COMMIT"
+merge_base=$(${merge_base_command})
+diff_command="git diff --name-only --diff-filter=ACM $current_branch_hash $merge_base"
```

Comment:
@jzoldak / Reviewers: FYI: One change I made beyond the review feedback with to add this `--diff-filter=ACM` so the script wouldn't attempt and fail when trying to lint deleted files.